client/src/pages/events/Form.js

```diff
@@ -0,0 +1,603 @@
+
+        const hostOrgSearchQuery = { status: Model.STATUS.ACTIVE }
+        const adminOrgSearchQuery = { status: Model.STATUS.ACTIVE }
+        const eventSeriesSearchQuery = {}
+
+        // Superusers can select parent organizations among the ones their position is administrating
+        if (!isAdmin) {
+          const orgsAdministratedUuids =
+            currentUser.position.organizationsAdministrated.map(org => org.uuid)
+          adminOrgSearchQuery.parentOrgUuid = [
+            currentUser.position.organization.uuid,
+            ...orgsAdministratedUuids
+          ]
+          adminOrgSearchQuery.orgRecurseStrategy = RECURSE_STRATEGY.CHILDREN
+
+          eventSeriesSearchQuery.adminOrgUuid = [
+            currentUser.position.organization.uuid,
+            ...orgsAdministratedUuids
+          ]
+        }
+        const locationFilters = Location.getReportLocationFilters()
+
+        const action = (
+          <>
+            <Button key="submit" variant="primary" onClick={submitForm}>
+              Save Event
+            </Button>
+            {notesComponent}
+          </>
+        )
+        const organizationFilters = {
+          allOrganizations: {
+            label: "All organizations",
+            queryVars: { status: Model.STATUS.ACTIVE }
+          }
+        }
+
+        const peopleFilters = {
+          allOrganizations: {
+            label: "All people",
+            queryVars: {
+              status: Model.STATUS.ACTIVE
+            }
+          }
+        }
+
+        const eventSeriesFilters = {
+          allEventSeries: {
+            label: "All event series",
+            queryVars: {
+              status: Model.STATUS.ACTIVE
+            }
+          }
+        }
+
+        const tasksFilters = {}
+
+        tasksFilters.allTasks = {
+          label: `All ${tasksLabel}`,
+          queryVars: { selectable: true }
+        }
+
+        if (values.hostOrg) {
+          tasksFilters.assignedToHostOrg = {
+            label: `Assigned to ${values.hostOrg.shortName}`,
+            queryVars: {
+              taskedOrgUuid: values.hostOrg.uuid,
+              orgRecurseStrategy: RECURSE_STRATEGY.PARENTS,
+              selectable: true
+            }
+          }
+        }
+
+        if (values.adminOrg && values.adminOrg.uuid !== values.hostOrg?.uuid) {
+          tasksFilters.assignedToAdminOrg = {
+            label: `Assigned to ${values.adminOrg.shortName}`,
+            queryVars: {
+              taskedOrgUuid: values.adminOrg.uuid,
+              orgRecurseStrategy: RECURSE_STRATEGY.PARENTS,
+              selectable: true
+            }
+          }
+        }
+
+        return (
+          <div>
+            <NavigationWarning isBlocking={dirty && !isSubmitting} />
+            <Messages error={saveError} />
+            <Form className="form-horizontal" method="post">
+              <Fieldset title={title} action={action} />
+              <Fieldset>
```

Comment:
Where is the `status` field?

Reply:
We might want to discuss what is the use case to decide events are active or inactive like People or Organizations, I dont think it makes any sense for events that happened in the past that you can go to edit them and make them inactive.  That means they do not show up anymore in searches?

Same for events in the future, if an event gets cancelled maybe we should have the option to delete them or have a "Cancelled" status. The status field, as it is used, for other entities does not really apply that well to events.